### PR TITLE
{math}[GCCcore, 9.3.0] Voro++ v0.4.6

### PR DESCRIPTION
**Pull request by ocaisa**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/cf25f5b323487ef8b4fe91883a6731c3 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/bbd154784d98832b623b954b21f75860 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/8962d2913b866504eb4df0bbcf6ec5ce for a full test report.

File: easybuild/easyconfigs/v/Voro++/Voro++-0.4.6-foss-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+to analyze a system of particles."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://math.lbl.gov/voro++/download/dir/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ef7970071ee2ce3800daa8723649ca069dc4c71cc25f0f7d22552387f3ea437e']
+
+# Let's store the checksum in order to be sure it doesn't suddenly change
```
Comment (akesandgren) [inline easybuild/easyconfigs/v/Voro++/Voro++-0.4.6-foss-2020a.eb]: Not sure why this comment is here....

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/855c97a684c44a17072026dc822d57fc for a full test report.

File: easybuild/easyconfigs/v/Voro++/Voro++-0.4.6-foss-2020a.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/v/Voro++/Voro++-0.4.6-foss-2020a.eb]: that seems to come from the time where checksum were not a regular thing :D

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b511c1add2139e9edddf6d94d7efd651 for a full test report.

File: easybuild/easyconfigs/v/Voro++/Voro++-0.4.6-intel-2020a.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Override PREFIX variable from Makefile
+installopts = "PREFIX=%(installdir)s"
+
+sanity_check_paths = {
+    'files': ['bin/voro++', 'lib/libvoro++.a', 'include/voro++/voro++.hh'],
+    'dirs': [],
+}
+
+modextrapaths = {'CPATH': 'include/voro++'}
+
```
Comment (lexming) [inline easybuild/easyconfigs/v/Voro++/Voro++-0.4.6-intel-2020a.eb]:
```suggestion
```

File: easybuild/easyconfigs/v/Voro++/Voro++-0.4.6-foss-2020a.eb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+# Override PREFIX variable from Makefile
+installopts = "PREFIX=%(installdir)s"
+
+sanity_check_paths = {
+    'files': ['bin/voro++', 'lib/libvoro++.a', 'include/voro++/voro++.hh'],
+    'dirs': [],
+}
+
+modextrapaths = {'CPATH': 'include/voro++'}
+
```
Comment (lexming) [inline easybuild/easyconfigs/v/Voro++/Voro++-0.4.6-foss-2020a.eb]:
```suggestion
```

**Comment by ocaisa** [comment]
Hmm, the toolchains used here is too high level from what I can tell, it just needs a C++ compiler. Since it is so old (last release was 2013) I'm leaning towards just putting it in GCCcore.

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/78744406bd1f2567c26a96729910356b for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/13f54f44bfea523096ab393fbd709eeb for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/d62098e9ab9a39abf06ee5c6143c71ca for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/8fab363adb8b0e0859f6bc28a86a281c for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/7d1de602428c97026c1d8418ce0d8980 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @ocaisa!

**Pull request merged**